resources/views/page/location/base.blade.php

```diff
@@ -1,14 +1,18 @@
 @extends('support::base')
 
 @section('content')
-    @include('locations::page.location.partials._identity_tag')
```

Comment:
Please explain -- why are you removing this?   It is required to satisfy unit testing validation.